packages/block-library/src/table-of-contents/style.scss

```diff
@@ -0,0 +1,4 @@
+:root :where(.wp-block-table-of-contents) {
```

Comment:
I just realised belatedly that this CSS selector should be simplified. The `:root :where()` combo makes the specificity the same as the simple class name and probably should just be `.wp-block-table-of-contents`.